kart/tabular/dataset3.py

```diff
@@ -160,7 +144,7 @@ def crs_definitions(self):
         """Returns {identifier: definition} dict for all CRS definitions in this dataset."""
         if not self.inner_tree or self.CRS_PATH not in self.inner_tree:
             return
-        for blob in find_blobs_in_tree(self.inner_tree / self.CRS_PATH):
+        for blob in find_blobs_in_tree(self.inner_tree / self.CRS_PATH, max_depth=4):
```

Comment:
why these extra kwargs? Feels unnecessary at best, and could introduce a nasty bug if we change the tree depth for some reason and don't realise

Reply:
I just moved this function into core and this was the old default since features were only ever at a quite a low max-depth compared to the feature tree. But I'll just take it out max-depth completely since it's more likely to cause a bug than to prevent it.